### DISCUSSION TITLE
Assets grafici

### DISCUSSION THREAD

**Discussion by guizzo** [Il-Libro-Open-Source/book#49] [Idee]
Ciao a tutti 👋 

Che dite, riusciamo a rendere un po' più cool ed invitante questa org (che già è fighissima così)? 😎 

Se magari tra di noi c'è qualcuno che ha voglia di contribuire realizzando le grafiche di avatar e banner per il profilo dell'org, mi offro volontario per creare il relativo repo ed impostare il tutto. 

Cosa ne pensate? 

@Cadienvan spero di aver aperto la discussione nel posto giusto; se così non fosse ti prego di farmelo sapere che sistemo subito

**Comment by serenasensini** [comment]
Io non ho competenze grafiche, ma mi accodo all'idea!

**Comment by Cadienvan** [comment]
Il posto è giusto e l'idea mi piace eccome!
Quindi pensavi di creare un repo a parte per la parte di graphics?
Vantaggi rispetto a tenere qui dentro il tutto?

P.S. Per le grafiche posso chiedere ad un amico, magari può aiutarci, lavora nel settore da quasi 30 anni.

**Reply by guizzo** [reply]
In realtà l'idea è quella di creare una sorta di "welcome page" dell'organizzazione, come ad esempio [questa](https://github.com/nodejs)

EDIT: Quindi non un repo per contenere gli assets grafici, ma un repo "speciale" offerto da Github dove puoi impostare ciò che verrà visualizzato nella home dell'organizzazione

EDIT 2: Ovviamente ci saranno da decidere i copy

**Reply by Cadienvan** [reply]
Aaaah ok capito, come le home dei profili che hanno il README.md, capito!

**Comment by Cadienvan** [comment]
Sarebbe figo farlo fare a Midjourney e similari, che ne pensate?

**Reply by guizzo** [reply]
Anche! Se pensi possa uscire fuori qualcosa di carino, una prova potremmo pure farla...alle brutte si scarta, no?

**Reply by Cadienvan** [reply]
Ho usato stability ai, non male:
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/08c6525d-dfba-43ba-8e2e-17a8e8297afb)
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/a64fa1c6-4c3f-4a3c-af3a-79075a9bcaf8)


Il prompt è stato il seguente, molto banale: A highly-detailed old-fashioned book with a natural light. The book should have an open source logo in the middle of the cover

**Reply by Cadienvan** [reply]
Variato il prompt (A highly-detailed book cover with a natural light and a transparent background. The cover should have some programming source code on the front and should represent freedom of speech), ecco i risultati:

![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/5f7e6885-356a-49d2-b79d-25221e6c1300)
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/8ec8f9ea-8e77-42ef-aa4b-1a362c10f1b5)
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/78b40a0e-20d5-4d9b-8461-7429a9736196)
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/df51a1e8-f12f-4e85-843e-d2a4c5439b29)
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/180aa8b7-a55b-4c61-bafb-0f00f91725e1)
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/500c6d4c-f516-4cc1-9875-25c36993f294)
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/76a7b6ee-f527-4e7d-9faa-9a1d11b442c0)

**Reply by guizzo** [reply]
Oh belle queste!

**Reply by Cadienvan** [reply]
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/7b53f0e5-9f77-4d99-87ec-f38d2987c934)
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/e742480c-855a-4aa8-b1ec-c4c3c2caeffe)
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/84bae486-185f-4233-8be2-be04cbef279b)
Mi sto innamorando.

**Reply by guizzo** [reply]
ok, abbiamo capito che le vogliamo in pixel art 😄

**Reply by Cadienvan** [reply]
![image](https://github.com/Il-Libro-Open-Source/book/assets/5492927/cb71f7fe-3e71-4ede-9e4c-9a8f80dc2a08)

Ultima perché mi piaceva troppo.

**Reply by guizzo** [reply]
Questa è bella bella, me gusta mucho

**Comment by Cadienvan** [comment]
There we go!
https://github.com/Il-Libro-Open-Source/.github

Basta ritoccare il README che sta nella cartella profile: https://github.com/Il-Libro-Open-Source/.github/blob/main/profile/README.md

**Reply by guizzo** [reply]
Mi hai battuto sul tempo, ero in attesa del via libera per crearlo io il repo 😂

**Reply by Cadienvan** [reply]
Sorry ahahah mi sono fatto prendere dal momento!

**Comment by guizzo** [comment]
@Cadienvan Ora però ci serve un'avatar...mica vorremmo lasciare quello brutto bruttissimo di default...nooooo

**Reply by Cadienvan** [reply]
Comincio a ritagliare il libro

**Reply by guizzo** [reply]
Più che altro questo io non posso farlo, non ho accesso ai settings dell'org

**Comment by guizzo** [comment]
@Cadienvan Questa direi di chiuderla, che dici?
